cmd/cluster-agent/api/agent/agent.go

```diff
@@ -20,7 +20,7 @@ import (
 	"github.com/DataDog/datadog-agent/cmd/agent/api/response"
 	"github.com/DataDog/datadog-agent/cmd/agent/common"
 	"github.com/DataDog/datadog-agent/cmd/agent/common/signals"
-	"github.com/DataDog/datadog-agent/cmd/cluster-agent/api/v1"
+	v1 "github.com/DataDog/datadog-agent/cmd/cluster-agent/api/v1"
```

Comment:
A nit here but why is it necessary to add an alias since it's already `v1`? Was it introduced by `gofmt`?

Reply:
Yep, `gofmt` has been executed on this file.